content/_changelogs/10.2.0.md

```diff
@@ -4,8 +4,10 @@ _Released 6/21/2022_
 
 **Features:**
 
-- Cypress is now available natively on Apple silicon (M1). Addresses
-  [#19908](https://github.com/cypress-io/cypress/issues/19908).
+- Cypress is now available natively on Apple silicon (M1). For more details,
```

Comment:
The blog mentions both M1 and M2 macs. Do we need to update to mention this or is the blog incorrect?

Reply:
I updated this with some verbiage from the blog post itself to call out both M1 and M2 support.